tdrs-backend/tdpservice/search_indexes/parsers/tanf_parser.py

```diff
@@ -22,33 +22,36 @@ def active_t1_parser(line, line_number):
         logger.error('Expected minimum line length of %s, got: %s', min_line_length, actual_line_length)
         return
 
-    for field in family_case_schema.get_all_fields():
-        if field.name == 'blank':
+    for field in family_case_schema:
+        print(f'Field: {field}')
```

Comment:
Was this print was just for dev/debugging? Should be removed or utilize the logger.